### PR TITLE
Implement can.getName symbol behavior

### PR DESCRIPTION
**Pull request by m-mujica**
Ref: https://github.com/canjs/can-symbol/issues/16

```js
// DefineMap
var Person = DefineMap.extend("Person", {});
var me = new Person();

me[canSymbol.for("can.getName")]() // "Person{1}"

// DefineList
var Things = DefineList.extend("Things", {});
var list = new Things([]);

list[canSymbol.for("can.getName")]() // "Things{2}"
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "can-observation": "^3.3.4",
     "can-reflect": "^1.2.1",
     "can-simple-observable": "^1.0.0",
-    "can-symbol": "^1.0.0",
+    "can-symbol": "canjs/can-symbol#friendly-name",
```
Comment (m-mujica) [inline package.json]: need to publish this first.

File: list/list.js [hunk 2]
```diff
@@ -1511,6 +1511,9 @@ canReflect.assignSymbols(DefineList.prototype,{
 	// get/set
 	"can.getKeyValue": DefineList.prototype.get,
 	"can.setKeyValue": DefineList.prototype.set,
+	"can.getName": function() {
+		return this.constructor.name + "[" + this._cid + "]";
```
Comment (matthewp) [inline list/list.js]: I think it's better to use `CID(this)` here, from can-cid, in case the implementation changes.

Comment (m-mujica) [reply list/list.js]: makes sense, will do.

**Comment by m-mujica** [comment]
going to add this for 4.0 instead.

**Pull request closed without merging**